ravenframework/Samplers/DynamicEventTree.py

```diff
@@ -152,6 +152,9 @@ def __init__(self):
     # variable representation (variable name) in the input file. For example,
     # {'var1':'var1,var2','var2':'var1,var2'}
     self.fullyCorrelatedEpistemicToVar     = {}
+    #
+    self.hybridConstants = {}
+    self.hybridDependentSample = {}
```

Comment:
(even if quite self explanatory, can you add a description of these variables?)

Reply:
fixed.